### DISCUSSION TITLE
网页速度[Bug]

### DISCUSSION THREAD

**Discussion by AndersonHJB** [vuepress-theme-hope/vuepress-theme-hope#3409] [Q&A] [answered]
### Checklist

- [X] I am using **LTS** version of Node.js.
- [X] I am using the **latest v2 version** of VuePress and VuePress official plugins.
- [X] I am using the **latest v2 version** of vuepress-theme-hope and all plugins in this repo.
- [X] I followed the docs and I double checked my configuration.
- [X] I have searched [FAQ](https://theme-hope.vuejs.press/faq/) and I did not find the answer.

### Package name

vuepress-theme-hope

### Which operating system are you using?

- [X] macOS
- [ ] Windows
- [ ] Linux

### Describe the bug

网站是部署在阿里云服务器上，其他页面都比较正常。开启了 全站 CDN
但是从主页切换到 Memoirs 有时候会很卡或者是常规很慢。

url：[https://bornforthis.cn/](https://bornforthis.cn/)

<img width="1280" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/48122016/f5696aa3-b5ba-448c-b604-7d1c5e397f41">

**Comment by AndersonHJB** [comment]
不知道是我自身的问题还是有优化的空间？

**Comment by Mister-Hope** [comment, marked as answer]
最新版本应该显著改进了此问题

**Reply by AndersonHJB** [reply]
非常期待，感谢

**Reply by Mister-Hope** [reply]
重新回顾了一下，这个性能问题难以避免，大量的标签和分类以及每个页面的摘要内容导致你的网站有：

1. 两个很庞大的分类、标签映射对应文章的ID
2. 一个巨大的加密配置（注意到你为很多文章设置了多个密码）
3. 一个更加巨大的文章信息数组，包含了每个文章的基本信息，如作者、分类、标签、时间、摘要等

上述内容约有 1MB，在解析中会浪费大量的时间，这是静态网站没法避免的问题。

我们已经避免了大量的性能问题，也就是如果你不进入博客页面，它们甚至都不会被解析（也就是打开普通页面相应很快），这也证明了我们性能的调优。

分站、禁用摘要等一定程度可以降低解析时间，即你可以考虑将你的文档站按照最大的分类拆成若干子站点，每个子站点速度会有显著提升。

**Comment by AndersonHJB** [comment]
因为在文章页的速度是比较正常和 ok 的，所以主页的卡顿等异常明显，辛苦了
